airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/jackson/MoreMappers.kt

```diff
@@ -16,6 +16,7 @@ import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
  * All jackson mapper creation should use the following methods for instantiation.
  */
 object MoreMappers {
```

Comment:
This whole class should be just companion object.

Reply:
see above